src/components/slice/Slice.tsx

```diff
@@ -0,0 +1,28 @@
+import * as React from 'react';
+import { StyleOf, BaseNodeProps, SceneNodeProps, LayoutProps, ExportProps } from '../../types';
+import {
+    LayoutStyleProperties,
+    transformLayoutStyleProperties
+} from '../../styleTransformers/transformLayoutStyleProperties';
+import { useYogaLayout } from '../../hooks/useYogaLayout';
+import { StyleSheet } from '../..';
+
+export interface SliceProps extends BaseNodeProps, SceneNodeProps, LayoutProps, ExportProps {
+    style?: StyleOf<LayoutStyleProperties>;
```

Comment:
I guess it should be `style?: StyleOf<LayoutStyleProperties & YogaStyleProperties>`;